bananagram-frontend/src/components/Grid/GridSquare.tsx

```diff
@@ -10,7 +10,7 @@ interface GridSquareProps {
     isCmdPressed: boolean
   ) => void;
 }
-
+// eslint-disable-next-line
```

Comment:
Do we need this eslint disable here?

Reply:
well mine moans about it

Reply:
What was it moaning about?

Reply:
![Screen Shot 2019-07-09 at 22 38 49](https://user-images.githubusercontent.com/25291974/60924990-66827600-a29a-11e9-85d0-0e934fee570c.jpg)
I don't see any errors here for some reason? Should be okay to remove this?

Reply:
Compiled with warnings.

./src/components/Grid/GridSquare.tsx
  Line 14:  Unexpected newline between template tag and template literal  no-unexpected-multiline